### DISCUSSION TITLE
Paraview Visualization of FEPX Postprocessed Results

### DISCUSSION THREAD

**Discussion by danishTUE** [neperfepx/neper#342] [Q&A] [answered]
Hi Romain,

I am interested to visualize (3D vizualisation) the results of FEPX in paraview. I try to do it by creating .vtk format file using Neper -V module (using -imageformat 'vtk') . However, when I run it, it always creates a .png file. I am interested to get a .vtk file which I can visualize in paraview in 3D.  For e.g, for example 4 of FEPX ("4_restart"), I am trying to visualise slip on a particular slip system in 3D. After running all the bash files, I try:"

```
neper -V cycle1-2.sim \
    -simstep 4 \
    -datanodecoo coo \
    -dataelt3dcol slip7 \
    -print vtk_try \
    -imageformat 'vtk'
```

However, it only creates .png file. Can you please suggest how can I visualise the FEPX results in paraview in 3D?

Many thanks in advance!

**Comment by rquey** [comment, marked as answer]
The order in which options are given is important in -V. Set the image format before printing it: `-imageformat vtk -print vtk_try`.

**Reply by danishTUE** [reply]
Thanks! That worked like a charm! Many thanks
